pkg/quota/v1/evaluator/core/resource_claims.go

```diff
@@ -99,10 +117,102 @@ func (p *claimEvaluator) MatchingResources(items []corev1.ResourceName) []corev1
 			strings.HasSuffix(string(item), corev1.ResourceClaimsPerClass /* by device class */) {
 			result = append(result, item)
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(features.DRAExtendedResource) {
+			if strings.HasPrefix(string(item), corev1.ResourceImplicitExtendedClaimsPerClass /* by implicit extended resource name */) ||
+				isExtendedResourceNameForQuota(item) /* by extended resource name */ {
```

Comment:
this will return true even for extended resources no DeviceClass is mapped to ... make sure we have a test that sets some unrelated extended resource in a quota to make sure this evaluator doesn't break when evaluating and is a no-op when actually evaluating